### PR TITLE
User service tests

### PR DESCRIPTION
**Pull request by IliaStepanov**

### CODE REVIEW

**Comment by IliaStepanov** [comment]
[[BE] UserService tests](https://trello.com/c/Bqx0a4EZ/95-be-userservice-tests)

File: src/test/java/com/epam/lowcost/services/implementations/UserServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    @Test
+    public void blockUser() {
+        User actualUser = new User();
+        User expectedUser = new User() {{
+            setActive(false);
+        }};
+        when(userRepository.findById((long) 1)).thenReturn(actualUser);
+        when(userRepository.save(actualUser)).thenReturn(actualUser);
+        userService.blockUser(1);
+        assertEquals(expectedUser, actualUser);
```
Comment (EugeneGrim) [inline src/test/java/com/epam/lowcost/services/implementations/UserServiceImplTest.java]: You don't check anything by this test.

File: src/test/java/com/epam/lowcost/services/implementations/UserServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        User actualUser = new User() {{
+            setActive(false);
+        }};
+        User expectedUser = new User() {{
+            setActive(true);
+        }};
+        when(userRepository.findById((long) 1)).thenReturn(actualUser);
+        when(userRepository.save(actualUser)).thenReturn(actualUser);
+        userService.unblockUser(1);
+        assertEquals(expectedUser, actualUser);
```
Comment (EugeneGrim) [inline src/test/java/com/epam/lowcost/services/implementations/UserServiceImplTest.java]: You don't check anything by this test.

File: src/test/java/com/epam/lowcost/services/implementations/UserServiceImplTest.java (hunk 1 above)
Comment (IliaStepanov) [reply src/test/java/com/epam/lowcost/services/implementations/UserServiceImplTest.java]: Made some changes we discussed.

File: src/test/java/com/epam/lowcost/services/implementations/UserServiceImplTest.java (hunk 2 above)
Comment (IliaStepanov) [reply src/test/java/com/epam/lowcost/services/implementations/UserServiceImplTest.java]: Made some changes we discussed.

**Pull request merged**